### PR TITLE
fix(fetch): handle in case some error happened in Fetch

### PR DESCRIPTION
**Pull request by safeamiiir**
A quick fix when fetch failed due to the issues with the file

### CODE REVIEW

**Review by jatinriverlane — APPROVED** [review]
Thanks @safeamiiir

File: aqueductcore/frontend/src/hooks/useUploadFile.ts [hunk 1]
```diff
@@ -53,6 +53,12 @@ function useFileUpload(experimentUuid: ExperimentDataType['uuid']) {
                         });
                 }
             })
+            // Somethings went wrong with the file
+            .catch((error) => {
+                toast.error(error.message ?? "Unable to upload file.", {
```
Comment (jatinriverlane) [inline aqueductcore/frontend/src/hooks/useUploadFile.ts:58]: The error message from the backend sends some relevant messages for some errors like Max Size Exceeded, Experiment Not Found, etc. I guess the error message can be shown here.

Comment (str-anger) [reply aqueductcore/frontend/src/hooks/useUploadFile.ts:58]: May we have also the file name as well? Because I might drop a batch.

Comment (safeamiiir) [reply aqueductcore/frontend/src/hooks/useUploadFile.ts]: file name added

Comment (safeamiiir) [reply aqueductcore/frontend/src/hooks/useUploadFile.ts]: @jatinriverlane That error is also handled in line 44

**Review by str-anger — APPROVED** [review]

**Pull request merged**